tests/components/homematicip_cloud/test_config_flow.py

```diff
@@ -55,18 +82,23 @@
         const.HMIPC_PIN: "123",
         const.HMIPC_NAME: "hmip",
     }
-    flow = config_flow.HomematicipCloudFlowHandler()
-    flow.hass = hass
-
-    with patch.object(
-        hmipc.HomematicipAuth, "async_setup", return_value=mock_coro(True)
-    ), patch.object(
-        hmipc.HomematicipAuth, "async_checkbutton", return_value=mock_coro(True)
-    ), patch.object(
-        hmipc.HomematicipAuth, "async_register", return_value=mock_coro(False)
+
+    with patch(
+        "homeassistant.components.homematicip_cloud.hap.HomematicipAuth.async_checkbutton",
+        return_value=True,
+    ), patch(
+        "homeassistant.components.homematicip_cloud.hap.HomematicipAuth.async_setup",
+        return_value=True,
+    ), patch(
+        "homeassistant.components.homematicip_cloud.hap.HomematicipAuth.async_register",
+        return_value=False,
     ):
-        result = await flow.async_step_init(user_input=config)
-        assert result["type"] == "abort"
+        result = await hass.config_entries.flow.async_init(
+            const.DOMAIN, context={"source": "user"}, data=config
+        )
+
+    assert result["type"] == "abort"
+    assert result["reason"] == "connection_aborted"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4887152)